pkg_resources/tests/test_resources.py

```diff
@@ -257,7 +257,7 @@ def test_marker_evaluation_with_extras(self):
             "/foo_dir/Foo-1.2.dist-info",
             metadata=Metadata((
                 "METADATA",
-                "Provides-Extra: baz\n" "Requires-Dist: quux; extra=='baz'",
+                "Provides-Extra: baz\nRequires-Dist: quux; extra=='baz'",
```

Comment:
Use triple-quoted strings instead?

Reply:
I think it is not necessary to use triple-quotes here, specially in the context of quick tests...

Reply:
But then tests are as important and should be as maintainable as the source code.

Reply:
I don't think that adding an explicit `\n` makes short snippets unreadable.
For tests, it even makes sense, because we are explicitly saying there should be a `\n` there...

On the other hand, using triple quotes for short strings either results in distracting code (e.g. when the indentation is broken inside the triple-quotes), or requires you to bring out the big guns (e.g. `inspect.cleandoc`).